### DISCUSSION TITLE
About how to quickly debug code?

### DISCUSSION THREAD

**Discussion by HCSR-Lab** [accel-sim/accel-sim-framework#164] [General] [answered]
Hi，respected person.
**Reason:** I'm experimenting with cache bypass and have a separate test for the cache part. But gdb is needed each time, to debug from scratch.
**Question:** Is it possible to directly debug cache-related code?

**Comment by JRPan** [comment, marked as answer]
Can you explain more? I'm not sure what you are exactly trying to do here. 

Generally, gdb is very good. You can add a print statement to print the variable if you want anything specific. You can also print other pipeline information. check here for more info https://github.com/accel-sim/accel-sim-framework/issues/128

**Reply by HCSR-Lab** [reply]
Thank you for your reply, it is very helpful for my experiment!
